drivers/scheduler/openshift/openshift.go

```diff
@@ -330,13 +332,66 @@ spec:
 	return err
 }
 
+// getImageSha get Imgae sha
+func getImageSha(ocpVersion string) (string, error) {
+	downloadURL := fmt.Sprintf("%s/%s/%s", OpenshiftMirror,
+		ocpVersion, mdFileName)
+	request := netutil.HttpRequest{
+		Method:   "GET",
+		Url:      downloadURL,
+		Content:  "application/json",
+		Body:     nil,
+		Insecure: true,
+	}
+	log.Infof("URL %s", downloadURL)
+	content, err := netutil.DoRequest(request)
+	if err != nil {
+		log.Errorf("Failed to get Get content from  %s, error %v", downloadURL, err)
+		return "", err
+	}
+	//Convert the body to type string
+	contentInString := string(content)
+	parts := strings.Split(contentInString, "\n")
+	for _, a := range parts {
+		if strings.Contains(a, "Image Digest:") {
+			return strings.Split(a, "`")[1], nil
+		}
+	}
+	return "", fmt.Errorf("Failed to find Image sha: in  %s", downloadURL)
+}
+
 func startUpgrade(upgradeVersion string) error {
 	var output []byte
 	var err error
-
+	var shaName string
 	args := []string{"adm", "upgrade", fmt.Sprintf("--to=%s", upgradeVersion)}
 	t := func() (interface{}, bool, error) {
 		if output, err = exec.Command("oc", args...).CombinedOutput(); err != nil {
```

Comment:
Add a debug log for what exact command you are running, will be easier to debug later when it fails. We might want to create our own `execCommand()` func that will use `exec.Command()`, but we can add all the verbosity and levels to it